bundles/ch.elexis.core.services/src/ch/elexis/core/services/OrderService.java

```diff
@@ -35,6 +36,12 @@ public IOrderEntry findOpenOrderEntryForStockEntry(IStockEntry stockEntry) {
 		return null;
 	}
 
+	public List<IOrderEntry> findOrderEntryForStock(IStock stock) {
+		IQuery<IOrderEntry> query = CoreModelServiceHolder.get().getQuery(IOrderEntry.class);
+		query.and("stockid", COMPARATOR.EQUALS, stock.getId());
```

Comment:
Statt dem String "stockid" solltest du hier `ModelPackage.Literals.ISTOCK_ENTRY__STOCK` verwenden, dann ist das Modellgebunden, bitte teste das mal ob das so funktioiert.

Reply:
Ich habe `ModelPackage.Literals.ISTOCK_ENTRY__STOCK` & `ModelPackage.Literals.IORDER_ENTRY__STOCK` versucht. Beide geben mir die Fehlermeldung aus: Could not resolve attribute...

Reply:
Ach meine Güte, du suchst ja in der Klasse `IOrderEntry`, entsprechend musst du auch die Literale aus der Klasse `ModelPackage.Literals.IORDER_ENTRY__STOCK` verwenden. Bitte probier das noch, eigentlich sollte der Literal sich dann auflösen. 

@huthomas oder ist `stockid` nicht über das Model erreichbar? Ich bin da grad puzzled!